tutorial/8 - expert.test.ts

```diff
@@ -104,29 +102,29 @@ describe.skip('expert', () => {
      */
     describe('`derivableCache`', () => {
         type Stocks = 'GOOGL' | 'MSFT' | 'APPL';
-        let stockPrice$: SinonStub<[Stocks], DerivableAtom<number>>;
-        beforeEach(() => stockPrice$ = stub<[Stocks], DerivableAtom<number>>().callsFake(() => atom.unresolved()));
+        let stockPrice$: jest.Mock<DerivableAtom<number>>;
+        beforeEach(() => stockPrice$ = jest.fn(() => atom.unresolved()));
 
-        const reactSpy = spy();
-        beforeEach(() => reactSpy.resetHistory());
-        function reactor(v: any) { reactSpy(v); }
+        const reactSpy = jest.fn(val => val);
+        beforeEach(() => reactSpy.mockClear());
+        // function reactor(v: any) { reactSpy(v); }
```

Comment:
Is this not necessary any more because of differences between sinon spies and jest mocks? If so, please remove the comment code